website/docs/plugin/which-sdk.mdx

```diff
@@ -41,6 +41,8 @@ should consider using the framework if you would benefit from the ability to:
 * Use nested attributes.
 * Use any type as the elements of a map.
 * Tell when an optional and computed field has been removed from a config.
+* Implement schema-based validation that is relative to the current attribute or applies to any list/map/set element.
+* Return warning or error diagnostics during resource destruction.
```

Comment:
Suggest rewording to "Return a warning"

Reply:
The intention here was to suggest that one or more of either warning and/or error diagnostics can be raised. My English here is a little fuzzy, but I think adding the article changes the meaning to imply only a single warning could be raised, which would not be accurate. Is there another suggestion you have here to clarify it? Maybe the wording could be switched around to something like "Return diagnostics of warning or error severity during resource destruction" although generally we just refer to them as "warning and error diagnostics"?